cat-test/lib/cat_cafe.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:math' show Random;
+
+// Let us feed your kitty!
+class CatCafe {
+  static final Random _index = new Random();
```

Comment:
`static final _index`. There's no need to type the left-hand side when it's the same as the type on the right-hand side.